src/Utilities.js

```diff
@@ -1,5 +1,7 @@
 // Copyright 2018, 2019 Stanford University see LICENSE for license
 
+import lookupConfig from '../static/spoofedFilesFromServer/fromSinopiaServer/lookupConfig.json'
```

Comment:
i'm uncomfy having code in production that is from a dir called "spoofedFilesFromServer" ... this feels like the wrong direction.  Move it to a non-spoof related directory and have the spoof read it from there?

Reply:
👍